client/src/components/App/Content/InvestigationForm/TabManagement/ExposuresAndFlights/Forms/ExposureForm/ExposureForm.tsx

```diff
@@ -47,6 +47,12 @@ const ExposureForm = (props: Props) => {
 		}
 	}, [exposureAndFlightsData.exposureSource]);
 
+	useEffect(() => {
+		if(exposureSourceSearchString === "") {
+			setOptionalCovidPatients([]);
+		}
+	}, [exposureSourceSearchString])
+
```

Comment:
same changes introduced in pr #1171
if they are relevant here -  maybe close #1171?
 or if they arent relevant here, i think they shouldn't be included here 🤔

Reply:
aw shit, must've branched from the other branch
Ill try to remove those changes from this branch

Reply:
reverted it [here](https://github.com/CoronaTeamMOD/CoronaInvestigation/pull/1172/commits/568b891332b1c87924c4cf7281cd1e139ad6e623)